### PR TITLE
Flag migrate emails for sawmill

### LINKED ISSUE #332: Indicate that sawmill should render a migration token email VS regular token login email
From https://github.com/mozilla/id.webmaker.org/issues/109

Need a method to indicate that sawmill should render a migration token email VS regular token login email for migrating users on id.webmaker.org.

### PR DESCRIPTION
**Pull request by cadecairos**
Fixes #332

### CODE REVIEW

**Comment by cadecairos** [comment]
R?

File: app/db/models/index.js [hunk 1]
```diff
@@ -315,14 +315,20 @@ module.exports = function (sequelize, env) {
           return callback(err);
         }
 
+        var loginUrlObj = url.parse(appURL, true);
+        loginUrlObj.search = null;
+        loginUrlObj.query.uid = userObj.getDataValue("username");
```
Comment (secretrobotron) [inline app/db/models/index.js]: much nicer :)

**Comment by secretrobotron** [comment]
Looks good. Should it come with a test?

**Comment by cadecairos** [comment]
> Should it come with a test?

normally, I'd say yes, but the current test setup doesn't have a way to test this part of the code, let alone the output to sawmill

**Comment by secretrobotron** [comment]
That's a shame. Looks alright to me though. Ship it.

**Pull request merged**